src/index.html

```diff
@@ -64,64 +53,111 @@ <h2><a class="anchor" id="game-rules"></a>Game Rules</h2>
+              If a robot knocks over a can, it loses <strong>1</strong> point.
           </ol>
+        <li>
+          If a robot has fewer than 0 points at the end of a match, it will be awarded 0 points.
         <li>
           At the end of the match the robot with the most points wins.
         <li>
-          The arena is a square of the design specified in
-          <a href="#specifications">the specifications section</a>.
+          SRO reserves the right to have match officials in the arena during matches.
+        <li>
+          The judge's decision is final.
+        <li>
+          Teams may not interact with their robot after the start of a match. This includes to restart their robot should it fail to start. Doing so may result in disqualification from the match.
+        <li>
+          While accidental bumps and scrapes are inevitable, the sport is non-contact.
+        <li>
+          Robots must not deliberately or negligently damage the arena or anything in it.
+        <li>
+          A robot will be stopped during a match when it is at risk of severely damaging itself, other robots, the arena, or otherwise poses a safety risk.
       </ol>
     </section>
     <section>
-      <h2><a class="anchor" id="regulations"></a>Regulations</h2>
+      <h2><a class="anchor" id="robot-design-rules"></a>Robot Design Rules</h2>
+      <p>
+        To ensure a fair competition, all teams must adhere to the following regulations.
+        Violations of these rules, without an exemption, may result in penalties.
+        These may include deductions of points or outright disqualification.
+      </p>
       <ol>
         <li>
-          All robots must be safe.
-        <li>
-          Decisions by match officials are final.
+          All robots must be <a href="#safety-regulations">safe</a>.
         <li>
           Teams must conduct themselves within the principles of good sporting conduct.
         <li>
-          Robots must be fully autonomous. No remote control systems or receivers are permitted.
+          Robots must fit within a 500 mm × 500 mm × 500 mm cube at the start of each match.
         <li>
-          Robots must pass a safety inspection before they can compete.
-          <ol>
-            <li>
-              Robots must not have any sharp edges or points.
-            <li>
-              Robots must have all electronics securely mounted.
-            <li>
-              Robots must provide adequate protection to the battery and its wires.
-            <li>
-              A robot's battery must be in the provided battery box, which must be securely mounted.
-            <li>
-              The power board must be mounted on the top of the robot, with its buttons easily accessible.
-          </ol>
+          During each match, robots must fit within a 600 mm × 600 mm horizontal square at all times. The robot's height is not restricted during the match.
```

Comment:
Issue on the height front: what do we do if a robot falls over? This was spotted at SR2025.

Reply:
Given this game, I'm going to just remove this point. Let's just keep a maximum size throughout.